lms/templates/calculator/toggle_calculator.html

```diff
@@ -140,4 +26,112 @@
       <input type="text" id="calculator_output" readonly />
     </form>
   </div>
-  </div>
+</div>
+
+<ul id="calculator_input_help" class="calc-help-popup" aria-hidden="true">
+  <li class="hint-item" id="hint-moreinfo">
+      <p><span class="bold">${_("For detailed information, see {math_link_start}Entering Mathematical and Scientific Expressions{math_link_end} in the {guide_link_start}edX Guide for Students{guide_link_end}.").format(
+        math_link_start='<a href="http://edx-guide-for-students.readthedocs.org/en/latest/SFD_mathformatting.html">',
+        math_link_end='</a>',
+        guide_link_start='<a href="http://edx-guide-for-students.readthedocs.org/en/latest/index.html">',
+        guide_link_end='</a>',
+      )}</span></p>
+  </li>
+
+  <li class="hint-item" id="hint-tips"><p><span class="bold">${_("Tips")}:</span> </p>
+    <ul>
+      <li class="hint-item" id="hint-paren"><p>${_("Use parentheses () to make expressions clear. You can use parentheses inside other parentheses.")}</p></li>
+      <li class="hint-item" id="hint-spaces"><p>${_("Do not use spaces in expressions.")}</p></li>
+      <li class="hint-item" id="hint-howto-constants"><p>${_("For constants, indicate multiplication explicitly (example: 5*c).")}</p></li>
+      <li class="hint-item" id="hint-howto-maffixes"><p>${_("For affixes, type the number and affix without a space (example: 5c).")}</p></li>
+      <li class="hint-item" id="hint-howto-functions"><p>${_("For functions, type the name of the function, then the expression in parentheses.")}</p></li>
+    </ul>
+  </li>
+
+  <li class="hint-item" id="hint-list">
+    <table class="calculator-input-help-table">
+      <tbody>
+        <tr>
+          <th scope="col">${_("To Use")}</th>
+          <th scope="col">${_("Type")}</th>
+          <th scope="col">${_("Examples")}</th>
+        </tr>
+        <tr>
+          <th scope="row">${_("Numbers")}</th>
+          <td dir="auto">${_("Integers")}<br />
+            ${_("Fractions")}<br />
+            ${_("Decimals")}
+          </td>
+          <td dir="auto">2520<br />
+            2/3<br />
+            3.14, .98
+          </td>
+        </tr>
+        <tr>
+          <th scope="row">${_("Operators")}</th>
+          <td dir="auto">
+            ## Translators: Please do not translate mathematical symbols.
+            ${_("+ - * / (add, subtract, multiply, divide)")}<br />
+
+            ## Translators: Please do not translate mathematical symbols.
+            ${_("^ (raise to a power)")}<br />
+
+            ## Translators: Please do not translate mathematical symbols.
+            ${_("|| (parallel resistors)")}
+          </td>
+          <td dir="auto">x+(2*y)/x-1
+            x^(n+1)<br />
+            v_IN+v_OUT<br />
+            1||2
+          </td>
+        </tr>
+        <tr>
+          <th scope="row">${_("Constants")}</th>
```

Comment:
Since we're here, it would be good to add a translator's comment, maybe even just
```This refers to symbols that are mathematical constants, such as "i" (square root of -1)```